scripts/coverage

```diff
@@ -7,4 +7,14 @@ fi
 
 set -x
 
+PYTHON_VERSION=$(${PREFIX}python -V)
+set +e
+echo $PYTHON_VERSION | grep "3.6" > /dev/null
+if [ $? -eq 0 ]; then
```

Comment:
May we rewrite it to something like

```suggestion
set +e
IS_PYTHON_LESS_37="$(${PREFIX}python -c 'import sys; print(sys.version_info[:2] < (3, 7))')"
if [[ ${IS_PYTHON_LESS_37} == "True" ]]; then
```

Benefits:
1. we control login using `python`, not `bash`
1. I'd rather use `[[` instead `[` ([reason](https://unix.stackexchange.com/questions/306111/what-is-the-difference-between-the-bash-operators-vs-vs-vs))

Reply:
:-) Looks like CI doesn't support `[[`

```console
Run scripts/coverage
+ python -c import sys; print(sys.version_info < (3, 7))
+ IS_BELOW_PY_3_7=True
+ [[ True == True ]]
scripts/coverage: 11: scripts/coverage: [[: not found
```